<launch>
 	<node name="talker" pkg="sound_play" type="soundplay_node.py" />

	<node name="asr_component" pkg="myrabot_dialog_b" type="asr_component.py" output="screen">
		<param name="dict" value="$(find myrabot_dialog_b)/demo/myrabot_demo1_10042014.dic" />
		<param name="fsg" value="$(find myrabot_dialog_b)/demo/myrabot_demo1_10042014.fsg"/>
	</node>
	
	<node name="voice_component" pkg="myrabot_dialog_b" type="voice_component.py" output="screen">
                <param name="voice" value="voice_don_diphone"/>
                <!-- <param name="wavepath" value="$(find myrabot_dialog_b)/sounds"/> -->
                <param name="voice_topic" value="/voice_component/voice_component_input"/>
        </node>
        
        <node name="dialog_component" pkg="myrabot_dialog_b" type="dialog_component.py" output="screen">
        </node>
 
        <node name="nlp_component" pkg="myrabot_dialog_b" type="nlp_component.py" output="screen">
                <param name="location" value="$(find dialog)/config/nlp_files/location_place.txt"/> 
                <param name="items" value="$(find dialog)/config/nlp_files/items.txt"/> 
                <param name="location_place" value="$(find dialog)/config/nlp_files/location_place.txt"/> 
                <param name="personal_noun" value="$(find dialog)/config/nlp_files/interaction.txt"/> 
                <param name="mode" value="$(find dialog)/config/nlp_files/mode.txt"/> 
                mode
        </node>
        
</launch>
